packages/bson-transpilers/test/yaml/bson-methods.yaml

```diff
@@ -176,6 +181,7 @@ tests:
             csharp: new MongoDBRef("coll", new ObjectId(), "db").DatabaseName
             shell: new DBRef('coll', new ObjectId(), "db").getDb()
             object: "'db'"
+            php: "['collectionName' => 'coll', 'id' => new \\MongoDB\\BSON\\ObjectId(), 'databaseName' => 'db']['databaseName']"
```

Comment:
Please correct the incorrect field names here. I'm not sure where you got these from, but [DBRefs](https://www.mongodb.com/docs/manual/reference/database-references/#dbrefs) in the MongoDB manual and the [DBRef specification](https://github.com/mongodb/specifications/blob/master/source/dbref.rst#dbref-structure) both specify what they should be.

Reply:
Fixed.

Reply:
Can you point me to the fix? I'm wondering if this line should now be updated in the PR.